build.sbt

```diff
@@ -2,7 +2,7 @@ name := "sct"
 
 organization := "com.github.bzumhagen"
 
-version := "0.8.1"
+version := "0.8.2"
```

Comment:
I think it's a big enough change to increment minor version to 0.9.0

Reply:
sure, agree